src/components/state-details.js

```diff
@@ -805,12 +753,14 @@ export default function StateDetailsPage ({ location, data }) {
       <hr className="mt-7" />
 
       <section className="text-center mb-8">
-        <div className="h1 mt-7 font-weight-bold">And that's it! 🎉</div>
+        <div className="h1 mt-7">✅</div>
+        <br className="d-none d-lg-block" />
+        <div className="h1 font-weight-bold">Ready to do your part?</div>
 
         <p className="h4 mt-4">
-          We hope this gives you some ideas for what you{" "}
+          Learn how to <strong>electrify your own machines</strong> and <strong>pass local policy</strong> to electrify the rest
           <br className="d-none d-lg-block" />
-          can do to get your state to zero emissions!
+          <a class='btn btn-lg btn-outline-secondary my-4' href='/take-action'>Take Action</a>
```

Comment:
Why do you use `<a>` here and `<Link>` on the homepage? I don't know enough about React to have a preference, but I think we should be consistent and pick one.

Reply:
`<Link>` is Gatsby's fast page navigation. Makes sense to use it wherever we can.